x-pack/plugins/security_solution/server/endpoint/routes/metadata/support/query_strategies.ts

```diff
@@ -77,42 +62,46 @@ export function metadataQueryStrategyV1(): MetadataQueryStrategy {
 export function metadataQueryStrategyV2(): MetadataQueryStrategy {
   return {
     index: metadataCurrentIndexPattern,
-    elasticAgentIdProperty: 'HostDetails.elastic.agent.id',
-    hostIdProperty: 'HostDetails.agent.id',
-    sortProperty: [
-      {
-        'HostDetails.event.created': {
-          order: 'desc',
-        },
-      },
-    ],
     extraBodyProperties: {
       track_total_hits: true,
     },
     queryResponseToHostListResult: (
-      searchResponse: SearchResponse<HostMetadata | HostMetadataDetails>
+      searchResponse: SearchResponse<HostMetadata | { HostDetails: HostMetadata }>
```

Comment:
this type is used in a few place - should you create a local `type` for it in this module and re-use it through out?

Reply:
I thought about that. I tried to make the intentional decision to delete the exported type, and not even keep that same type(ish) as internal, but do it this way to make it extremely clear this is ephemeral, and the closest functions to the data will strip this prefix out. So no other space could imagine to deal with it